starfish/intensity_table.py

```diff
@@ -276,3 +278,54 @@ def synthetic_intensities(
         intensities[cls.Constants.GENE.value] = ('features', genes)
 
         return intensities
+
+    @classmethod
+    def from_image_stack(cls, image_stack, crop: Tuple[int, int, int]=(0, 0, 0)) -> "IntensityTable":
```

Comment:
This sort of thing might bite us in the long run.  Is there a compelling reason to do a single crop variable that contains all the dimensions?  Why not pass in crop_x, crop_y, and crop_z individually?

Reply:
Rationale: it is easy to mess up the order of the tuple.

Reply:
Got it, now I understand. That makes sense. I'll make this change.

Reply:
If you want to be really daring, you could stick it after a '*' parameter, which would require callers to explicitly name the parameters. 

`from_image_stack(...., 0, 15, 12)` is less readable than `from_image_stack(...., crop_x=0, crop_y=15, crop_z=12)`